integration/shared/isolated/marketplace_command_test.go

```diff
@@ -158,9 +158,9 @@ var _ = Describe("marketplace command", func() {
 						Eventually(session).Should(Say("Getting services from marketplace in org %s / space %s as %s\\.\\.\\.", org2, space2, user))
 						Eventually(session).Should(Say("OK"))
 						Eventually(session).Should(Say("\n\n"))
-						Eventually(session).Should(Say("service\\s+plans\\s+description"))
+						Eventually(session).Should(Say("service\\s+plans\\s+description\\s+broker"))
```

Comment:
Also, as you do this, be aware that you might need to assert different things depending on the version of the CC the integration tests are being run on. All of our integration tests run against a legacy environment (as of Jan 2019, this is 1.7.0 of cf-deployment). As an example, see these tests: https://github.com/cloudfoundry/cli/blob/83e10ced7beac0a02f9070ea1e8b3abb026cff21/integration/v6/isolated/app_command_test.go#L307

On a related note, if someone is running the CLI built after this PR with a CF version that doesn't have `service_broker_name`, what does the output look like? Or has `service_broker_name` been in there for a long time?

Reply:
In the integration tests for marketplace, this is the only place where we would see the broker name, the other happy path is with the `-s`, which does not show the broker column, more details of it are in [this story](https://www.pivotaltracker.com/n/projects/2105761/stories/162647822).

Reply:
We've updated the PR, and separated the tests for broker column based on CC api version check.  CC api has the broker name in the response for sometime now(Oct'18), but the CLI is aware of it only through this commit. For someone running the CC API below 2.125, it would basically have an empty broker column.